src/content/docs/nosecone/quick-start.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "Arcjet Nosecone quick start"
+description: "Quick start guide for applying secure headers with Nosecone."
+prev: false
+next: false
+frameworks:
+  - bun
+  - deno
+  - next-js
+  - node-js
+  - sveltekit
+ajToc:
+  - text: 1. Install Nosecone
+    anchor: "1-install-nosecone"
+  - text: 2. Configure your application
+    anchor: "2-configure-your-application"
+  - text: 3. Run your application
+    anchor: "3-run-your-application"
+  - text: 4. Inspect the headers
+    anchor: "4-inspect-the-headers"
+---
+
+import WhatIsArcjet from "/src/components/WhatIsArcjet.astro";
+import SlotByFramework from "@/components/SlotByFramework";
+
+import BunStep1 from "@/snippets/nosecone/quick-start/bun/Step1.mdx";
+import BunStep2 from "@/snippets/nosecone/quick-start/bun/Step2.mdx";
+import BunStep3 from "@/snippets/nosecone/quick-start/bun/Step3.mdx";
+import BunStep4 from "@/snippets/nosecone/quick-start/bun/Step4.mdx";
+
+import DenoStep1 from "@/snippets/nosecone/quick-start/deno/Step1.mdx";
+import DenoStep2 from "@/snippets/nosecone/quick-start/deno/Step2.mdx";
+import DenoStep3 from "@/snippets/nosecone/quick-start/deno/Step3.mdx";
+import DenoStep4 from "@/snippets/nosecone/quick-start/deno/Step4.mdx";
+
+import NextJsStep1 from "@/snippets/nosecone/quick-start/next-js/Step1.mdx";
+import NextJsStep2 from "@/snippets/nosecone/quick-start/next-js/Step2.mdx";
+import NextJsStep3 from "@/snippets/nosecone/quick-start/next-js/Step3.mdx";
+import NextJsStep4 from "@/snippets/nosecone/quick-start/next-js/Step4.mdx";
+
+import NodeJsStep1 from "@/snippets/nosecone/quick-start/node-js/Step1.mdx";
+import NodeJsStep2 from "@/snippets/nosecone/quick-start/node-js/Step2.mdx";
+import NodeJsStep3 from "@/snippets/nosecone/quick-start/node-js/Step3.mdx";
+import NodeJsStep4 from "@/snippets/nosecone/quick-start/node-js/Step4.mdx";
+
+import SvelteKitStep1 from "@/snippets/nosecone/quick-start/sveltekit/Step1.mdx";
+import SvelteKitStep2 from "@/snippets/nosecone/quick-start/sveltekit/Step2.mdx";
+import SvelteKitStep3 from "@/snippets/nosecone/quick-start/sveltekit/Step3.mdx";
+import SvelteKitStep4 from "@/snippets/nosecone/quick-start/sveltekit/Step4.mdx";
+
+Arcjet Nosecone library provides a streamlined way to apply secure headers to
+your application.
+
+<WhatIsArcjet />
```

Comment:
I suggest we remove this component and have a new one which explains that Arcjet is providing this free utility, but it doesn't require you to use Arcjet. We can do the same on the Redact page, too.